core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -1484,7 +1484,7 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
     uint32 cond, count, fidx, tidx, frame_size = 0;
     uint32 all_cell_num = 0;
     int16 addr1, addr2, addr_ret = 0;
-    int32 didx, val;
+    int32 i, didx, val;
```

Comment:
L5795 also declares variable i, it will be shadowed.
How about putting this variable in L5907, and declaring it as uint32:
```C
    uint32 cell_num_of_local_stack;
#if WASM_ENABLE_REF_TYPES != 0 && WASM_ENABLE_GC == 0
    uint32 i;
#endif
```